src/overlays/actors/ovl_Bg_Astr_Bombwall/z_bg_astr_bombwall.c

```diff
@@ -15,7 +16,15 @@ void BgAstrBombwall_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgAstrBombwall_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgAstrBombwall_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-#if 0
+void BgAstrBombwall_InitCollider(ColliderTrisInit* init, Vec3f* pos, Vec3s* rot, ColliderTris* collider);
+void func_80C0A120(BgAstrBombwall* this, GlobalContext* globalCtx);
+void func_80C0A378(BgAstrBombwall* this);
+void func_80C0A38C(BgAstrBombwall* this, GlobalContext* globalCtx);
+void func_80C0A400(BgAstrBombwall* this, GlobalContext* globalCtx);
+void func_80C0A418(BgAstrBombwall* this, GlobalContext* globalCtx);
+void func_80C0A458(BgAstrBombwall* this, GlobalContext* globalCtx);
+void func_80C0A4BC(BgAstrBombwall* this, GlobalContext* globalCtx);
```

Comment:
We only include necessary forward declarations up here, you can remove at least one of these.